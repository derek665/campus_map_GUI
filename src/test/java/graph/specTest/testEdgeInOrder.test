# test if edges are in order when printed

# Create graph
CreateGraph alp2

# Add nodes
AddNode alp2 n1
AddNode alp2 n2

# Add edges
AddEdge alp2 n1 n2 a1
AddEdge alp2 n1 n2 a2
AddEdge alp2 n1 n2 b2
AddEdge alp2 n1 n2 b1

# Print the edges
ListChildren alp2 n1