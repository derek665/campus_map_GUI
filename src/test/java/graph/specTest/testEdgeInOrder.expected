# test if edges are in order when printed

# Create graph
created graph alp2

# Add nodes
added node n1 to alp2
added node n2 to alp2

# Add edges
added edge a1 from n1 to n2 in alp2
added edge a2 from n1 to n2 in alp2
added edge b2 from n1 to n2 in alp2
added edge b1 from n1 to n2 in alp2

# Print the edges
the children of n1 in alp2 are: n2(a1) n2(a2) n2(b1) n2(b2)